src/librustc_resolve/lib.rs

```diff
@@ -257,7 +257,7 @@ fn resolve_struct_error<'b, 'a: 'b, 'c>(resolver: &'b Resolver<'a>,
                              "type `{}` is not a member of trait `{}`",
                              type_,
                              trait_);
-            err.span_label(span, &format!("not a member of trait `Foo`"));
+            err.span_label(span, &format!("not a member of `{}`", trait_));
```

Comment:
Okay, I can add "trait" to all three labels. Wish Github had a way to squash commits in the web interface...